adapters/vidazoo/params_test.go

```diff
@@ -2,8 +2,9 @@ package vidazoo
 
 import (
 	"encoding/json"
-	"github.com/prebid/prebid-server/v2/openrtb_ext"
 	"testing"
+
```

Comment:
Noting +2, -1 changed lines here that are unrelated to the version package update.